.github/workflows/tests.yml

```diff
@@ -141,7 +141,7 @@ jobs:
       - name: Download coverage data
         uses: actions/download-artifact@v4
         with:
-          name: coverage-data
+          name: coverage-data-ubuntu-latest-3.10
```

Comment:
No need to hardcode. Use `cruft update` to update template with fixes for this, Or look at https://github.com/statisticsnorway/ssb-pypitemplate-instance/blob/main/.github/workflows/tests.yml#L143-L147